### PR TITLE
build cdc for multiarch

### PR DESCRIPTION
**Pull request by asottile-sentry**
this will push an app image to ghcr, built for both amd64 and arm64 for use with the sentry dev tools

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -3,5 +3,5 @@ PyYAML==5.1
 confluent-kafka==0.11.6
 datadog==0.21.0
 jsonschema==3.0.1
-psycopg2-binary==2.7.7
+psycopg2-binary==2.9.3
```
Comment (joshuarli) [inline requirements.txt:6]: intentional?

**Review by joshuarli — APPROVED** [review]

File: requirements.txt (hunk 1 above)
Comment (asottile-sentry) [reply requirements.txt:6]: yeah the older version does not build on arm64

File: Dockerfile [hunk 2]
```diff
@@ -58,7 +58,13 @@ RUN mkdir -p /usr/src/app
 WORKDIR /usr/src/app
 
 COPY requirements.txt /usr/src/app/
-RUN pip install -r requirements.txt
+RUN set -x \
+    && deps='gcc librdkafka-dev libc-dev' \
```
Comment (fpacifici) [inline Dockerfile:62]: Why do we need this ?

**Review by fpacifici — APPROVED** [review]
Are you trying to make our fork of wal2json multi arch as well ?
https://github.com/getsentry/wal2json

File: Dockerfile (hunk 2 above)
Comment (asottile-sentry) [reply Dockerfile:62]: confluent-kafka doesn't have a prebuilt on arm64 sadly

**Comment by asottile-sentry** [comment]
> Are you trying to make our fork of wal2json multi arch as well ? https://github.com/getsentry/wal2json

hmmm!  wasn't aware of this but if I run into it I'll fix it too !

**Pull request merged**